lib/types.ts

```diff
@@ -65,7 +65,8 @@ export type FactType =
   | "imageSizeBytes"
   | "loadedPackages"
   | "imageCreationTime"
-  | "redHatRepositories";
+  | "redHatRepositories"
+  | "imageNames";
```

Comment:
We had a super lengthy discussion about this as well 😆 
We settled on image names for a few reasons:
- OCI does not actually define a name for this...
- All other vendors in this space call the full reference an "image name" - we went with this for consistency

I also wanted to call it "image reference" but in the end we chose to be consistent - both the internal and public naming for this is "image name" to not cause confusion. 

IMO we can still call it reference here but it will be confusing eventually to have 2 different names internally for the same concept! Docker plugin would call it "reference" but anywhere in our backend it will be "image name"

Reply:
We had this exact same discussion before when designing the image model and came across the same pain points. Originally it was called `imageReference`, until we found the definition of `reference` in the OCI image spec. As you can see in the [Miro board](https://miro.com/app/board/o9J_ltv1rBg=/) here. In the end, as most of the competitors (as you can see in the miro board too) including customers refer to the whole `nginx:latest` as an image name, hence we picked that 😅

Reply:
😆 fun!

Alright, I definitely agree that consistency is more important here than whatever opinion I (or Ivan ;) ) have about the seemingly correct name for this 😄 

I'm fine with `imageName`, let's keep it like that then.